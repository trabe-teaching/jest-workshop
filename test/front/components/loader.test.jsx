import React from "react";
import { render } from "enzyme";
import Loader from "components/loader";

describe("front/components/loader", () => {
  it("renders nothing if not visible", () => {
    expect(render(<Loader visible={false} />)).toMatchSnapshot();
  });

  it("renders if visible", () => {
    expect(render(<Loader visible />)).toMatchSnapshot();
  });
});
