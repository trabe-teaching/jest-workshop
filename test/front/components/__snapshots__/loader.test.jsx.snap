// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`front/components/loader renders if visible 1`] = `
<p>
  Loading
</p>
`;

exports[`front/components/loader renders nothing if not visible 1`] = `null`;
